app/controllers/agendapoints/edit.js

```diff
@@ -89,6 +85,7 @@ export default class AgendapointsEditController extends Controller {
       editorDocument.updatedOn = new Date();
       editorDocument.title = this.editorDocument.title;
       editorDocument.previousVersion = this.editorDocument;
+      editorDocument.documentContainer = this.documentContainer;
```

Comment:
this might get lost in the big lump, but this is a bug that will need a migration to correct on all environments. There's supposed to be a direct link between any version and the container and that was no longer being set.